lua/solarized-osaka/theme.lua

```diff
@@ -835,6 +853,12 @@ function M.setup()
       end
     end
   end
+  local markdown_rainbow = { c.blue, c.yellow, c.green, c.red, c.magenta, c.cyan }
```

Comment:
I don't understand what these changes are for.

Reply:
These are for markdown headings and providing different colors for each of them.
![Screenshot from 2024-03-12 08-20-53](https://github.com/craftzdog/solarized-osaka.nvim/assets/13983258/4c5b4f80-c159-4903-8bbd-3cf7c57a7985)

Before all of them were red so there was no differentiation between `H1` and `H2` when editing unless you look at the signs being shown beside them.

![Screenshot from 2024-03-12 08-21-50](https://github.com/craftzdog/solarized-osaka.nvim/assets/13983258/351a8011-aefb-4d2e-b3cc-8c58db9b1abf)

Reply:
okay thanks. but I don't really like colorful headings. you can use them in your config.
Could we just have the darker background color for them?

Reply:
Removed back to how it was with a dark background as in the before screenshots.